state/state.go

```diff
@@ -42,6 +43,16 @@ func (state *State) SetManager(obj interface{}) error {
 	return nil
 }
 
+func (state *State) SetOutput(obj interface{}, outputName string) error {
+	_, err := state.configJSON.SetP(obj, fmt.Sprintf("output.%s", outputName))
+	if err != nil {
+		return err
+	}
+	ioutil.WriteFile("/tmp/main.tf.json", state.configJSON.Bytes(), 0777)
```

Comment:
This should take some steps to prevent hijacking.

E.g., this is a hard coded location. An attacker could pre-create a symlink to an arbitrary file on disk, either overwriting it or causing an error on write.

Possible alternatives:

* Use a random path name (e.g., similar to the `mktemp` shell command)
* Use a location inside the user's home directory

Reply:
What about this one? Can we use a random name or something inside the user's homedir? Also this file probably should not be world writable.

Reply:
I have just made a change on this.

This was not necessary. I have just removed this line. :)